instrumentation/lettuce/lettuce-5.1/testing/src/main/java/io/opentelemetry/instrumentation/lettuce/v5_1/AbstractLettuceClientTest.java

```diff
@@ -72,4 +79,12 @@ private ContainerConnection(StatefulRedisConnection<String, String> connection,
       this.port = port;
     }
   }
+
+  protected static List<AttributeAssertion> addExtraAttributes(AttributeAssertion... assertions) {
+    List<AttributeAssertion> result = new ArrayList<>(Arrays.asList(assertions));
+    if (Boolean.getBoolean("testLatestDeps")) {
+      result.add(equalTo(DbIncubatingAttributes.DB_NAMESPACE, "0"));
```

Comment:
@trask we use lettuce built in tracing and that uses `db.namespace`, should we map it to `db.name` when using old semconv? This attribute is only generated by the latest lettuce.